### PR TITLE
NEWS: News file sync with v1.4.0 tree

### PR DESCRIPTION
**Pull request by shamisp**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7971/ for details (Mellanox internal link).

File: NEWS [hunk 1]
```diff
@@ -44,6 +47,12 @@ Tested configurations:
 - KNEM: 1.1.2
 - Multiple bugfixes (full list on github)
 
+Known issues:
+  #2919 - Segfault in CUDA support when KNEM not present and CMA is active
+  intra-node RMA transpor. As a workaround user can disable CMA support at
```
Comment (miked-mellanox) [inline NEWS]: typo: transport

Comment (shamisp) [reply NEWS]: Thanks. I will fix

**Comment by shamisp** [comment]
@miked-mellanox please approve. thanks

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7993/ for details (Mellanox internal link).

**Pull request merged**